### PR TITLE
TVB-2692: Fixed problem with Kuramoto model

### PR DESCRIPTION
**Pull request by VinczeRobert**
Luckily, the issue was just the fact that a line `"of=str"` was missing from this model, sadly it requires a change in the library and I know we don't really like that.

As a so called bonus, I have noticed the Janset Rit and Zerlaut were having the same problem so I fixed those too and the Janset Rit model were also not having its variables of interest as a MultiSelectField so I changed that as well.

### CODE REVIEW

File: scientific_library/tvb/simulator/models/jansen_rit.py [hunk 1]
```diff
@@ -459,6 +459,7 @@ class ZetterbergJansen(Model):
         it is also provides the default range of phase-plane plots.""")
 
     variables_of_interest = List(
+        of=str,
```
Comment (liadomide) [inline scientific_library/tvb/simulator/models/jansen_rit.py:462]: These are changes ok to be made even if they are in the library. 
They are left-ver bugs from our neotraits migration.

**Comment by VinczeRobert** [comment]
> I am ok with the changes in the models, as long as all the previous running unit-tests will pass.
> 
> But I right to say that changes unrelated to this bug are also included in this branch?
> Could pls cleanup the history ?

Done.

**Comment by liadomide** [comment]
EpileptorCodim3SlowModModelForm
`self.variables_of_interest = ArrayField(ModelsEnum.EPILEPTOR_CODIM_3_SLOW`
Isn't this also a problem ?

**Comment by VinczeRobert** [comment]
It is. I'll fix it immediately.

**Comment by VinczeRobert** [comment]
I am done. The fields are good now for each model.

**Pull request merged**